### ISSUE TITLE
Drop the persistence workarounds once the Fennec persistence fixes are released

### ISSUE THREAD

**Issue by maho7791** [eclipse-fennec/emf.ogc.features#9]
The persistence issues found while building this project (eclipse-fennec/emf.persistence-jpa#312–#316, follow-ups #319/#320) are fixed or about to be. This lists what can then go here. Suggestions only — each item names the persistence change it depends on.

## Available in the current persistence snapshot (`db3b9f7`)

**1. `DemoDataLoader`: no virtual thread, no retry** — eclipse-fennec/emf.persistence-jpa#313 (PR #318)
The unit was never unavailable; the `jpa://` resource factory had not yet been told about it. It now resolves the unit from the service registry, so the loader can load in `@Activate` directly:

```java
@Activate
public DemoDataLoader(...) {
    ...
    load();
    registerCondition();
}
```

This also takes one suspect out of #8.

**2. Javadoc of `OgcFeaturesAnnotations.PROPERTY`** — eclipse-fennec/emf.persistence-jpa#314 (PR #317)
ExtendedMetaData names are no longer table/column names unless the mapping sets `fennec.jpa.eorm.useNamesFromExtendedMetaData=true`. The own `property` annotation can stay, but the reason given ("Fennec persistence also takes them as column names") is outdated. A name that is no SQL identifier now fails the unit deploy with a diagnostic naming the attribute instead of breaking the DDL silently.

## After eclipse-fennec/emf.persistence-jpa#321 is merged and released

**3. `GeoJsonTypeConverter` declares large values; `TextColumns` and `widenColumns` go** — eclipse-fennec/emf.persistence-jpa#319
`TypeConverter` gets `isLargeValue(EClassifier)` (default `false`). A converter answering `true` gets its String column mapped as a Lob (`TEXT` on PostgreSQL, CLOB on H2) unless the mapping declares a length or Lob itself:

```java
@Override
public boolean isLargeValue(EClassifier eDataType) {
    return isConverterForType(eDataType); // a geometry as GeoJSON text has no useful bound
}
```

Then `TextColumns.widen` (the `ALTER` through the DataSource after schema generation), the `widenColumns` config and the loader's `dataSource` reference are no longer needed. That is the second suspect in #8. Per attribute, `lob=true` or `length=…` on the `https://eclipse.org/fennec/persistence` annotation works too.

## Depends on #8

**4. `DB_CLOSE_DELAY=-1` in `config/datasource-h2.json`** — eclipse-fennec/emf.persistence-jpa#316
Keep it until #8 is resolved. Independently of the data loss, without a pool EclipseLink holds no connection between operations, so an embedded H2 with `DB_CLOSE_DELAY=0` is closed and reopened around every operation; `-1` is the sensible setting for the demo anyway (now in the persistence JPA user guide).

**Comment by maho7791** [comment]
eclipse-fennec/emf.persistence-jpa#321 is merged — item 3 (`TypeConverter.isLargeValue`, the `lob`/`length` annotation keys) is available with the next persistence snapshot build.
